public/js/main.js

```diff
@@ -76,7 +76,9 @@ function getTargetFromQuery() {
 
 const connTarget = getTargetFromQuery();
 if (connTarget) {
-  startDebugging(connTarget, actions).then(() => {
+  startDebugging(connTarget, actions).then((tabs) => {
+    actions.newTabs(tabs);
```

Comment:
noticed that the tabs and selectedTab state were not being set when the target was in the query